bricks/test_optimizer/hooks/pubspec.yaml

```diff
@@ -8,7 +8,10 @@ dependencies:
   mason: ">=0.1.0-dev.41 <0.1.0"
   path: ^1.8.1
 
+
+# Beware: on hooks, even dev dependencies have to be compatible to all dart versions covered by
```

Comment:
Is there a way we can automate checking this?

Reply:
E2e testing of the test command

Reply:
Added https://github.com/VeryGoodOpenSource/very_good_cli/issues/676 for this